src/Client/Pages/Home.fs

```diff
@@ -25,16 +24,21 @@ let view () =
                     [
                         Column.column [ ] [
                             div [ ClassName "home-link-container" ] [
-                                Markup.viewLinkCentered "home-column-links link-block" Page.NounDeclension (linkBlock "Noun Declension" "noun-color")
-                                Markup.viewLinkCentered "home-column-links link-block" Page.AdjectiveDeclension (linkBlock "Adjective Declension" "adjective-color")
-                                Markup.viewLinkCentered "home-column-links link-block" Page.AdjectiveComparatives (linkBlock "Adjective Comparative" "adjective-color")
+                                Markup.viewLinkCentered "home-column-links link-block" Page.NounDeclension (linkBlock "Noun Declension" "noun-color" "N")
+                                Markup.viewLinkCentered "home-column-links link-block" Page.NumeralCardinals (linkBlock "Numerals Cardinals" "numeral-color" "123")
+                                Markup.viewLinkCentered "home-column-links link-block" Page.AdjectiveDeclension (linkBlock "Adjective Declension" "adjective-color" "A")
                             ]
                         ]
                         Column.column [ ] [
                             div [ ClassName "home-link-container" ] [
-                                Markup.viewLinkCentered "home-column-links link-block" Page.VerbImperatives (linkBlock "Verb Imperative" "verb-color")
-                                Markup.viewLinkCentered "home-column-links link-block" Page.VerbParticiples (linkBlock "Verb Participle" "verb-color")
-                                Markup.viewLinkCentered "home-column-links link-block" Page.VerbConjugation (linkBlock "Verb Conjugation" "verb-color")
+                                Markup.viewLinkCentered "home-column-links link-block" Page.AdjectiveComparatives (linkBlock "Adjective Comparative" "adjective-color" "A")
+                                Markup.viewLinkCentered "home-column-links link-block" Page.VerbImperatives (linkBlock "Verb Imperative" "verb-color" "V")
+                                Markup.viewLinkCentered "home-column-links link-block" Page.VerbParticiples (linkBlock "Verb Participle" "verb-color" "V")
+                            ]
+                        ]
+                        Column.column [ ] [
+                            div [ ClassName "home-link-container" ] [
+                                Markup.viewLinkCentered "home-column-links link-block" Page.VerbConjugation (linkBlock "Verb Conjugation" "verb-color" "V")
                             ]
                         ]
```

Comment:
I would try this way:
```fsharp
Column.column [ ] [
    div [ ClassName "home-link-container" ] [
        Markup.viewLinkCentered "home-column-links link-block" Page.NounDeclension (linkBlock "Noun Declension" "noun-color" "N")
        Markup.viewLinkCentered "home-column-links link-block" Page.AdjectiveComparatives (linkBlock "Adjective Comparative" "adjective-color" "A")
        Markup.viewLinkCentered "home-column-links link-block" Page.AdjectiveDeclension (linkBlock "Adjective Declension" "adjective-color" "A")
    ]
]
Column.column [ ] [
    div [ ClassName "home-link-container" ] [
        Markup.viewLinkCentered "home-column-links link-block" Page.VerbConjugation (linkBlock "Verb Conjugation" "verb-color" "V")
        Markup.viewLinkCentered "home-column-links link-block" Page.VerbImperatives (linkBlock "Verb Imperative" "verb-color" "V")
        Markup.viewLinkCentered "home-column-links link-block" Page.VerbParticiples (linkBlock "Verb Participle" "verb-color" "V")
    ]
]
Column.column [ ] [
    div [ ClassName "home-link-container" ] [
        Markup.viewLinkCentered "home-column-links link-block" Page.NumeralCardinals (linkBlock "Numerals Cardinals" "numeral-color" "123")
    ]
]
```

As a result:
![image](https://user-images.githubusercontent.com/5451366/96188217-a248c880-0f3e-11eb-88bb-23d37d7d9d33.png)

Looks more aesthetic to me, but it's probably subjective.